__tests__/app.test.js

```diff
@@ -259,5 +259,73 @@ describe('app', () => {
     })
 })
 })
+})
+    describe('PATCH/api/reviews/:review_id', () => {
+        test('200: increment: responds with updated votes in review', () => {
+            return request(app)
+            .patch('/api/reviews/2')
+            .send({inc_votes: 5})
+            .expect(200)
+            .then(({body}) => {
+                const updatedReview = body
+                expect (updatedReview.votes).toBe(10)
+            })
+        });
+        test('200: decrement: responds with updated votes in review, decerement', () => {
+            return request(app)
+            .patch('/api/reviews/4')
+            .send({inc_votes: -4})
+            .expect(200)
+            .then(({body}) => {
+                const updatedReview = body
+                expect (updatedReview.votes).toBe(3)
+            })
+        });
+        test('200: only a body containing the key of inc_votes can be inputted', () => {
+            return request(app)
+            .patch('/api/reviews/2')
+            .send({
+                inc_votes: 5,
+                new_name: 'the avatar'
+            })
+            .expect(200)
+            .then(({body}) => {
+                const updatedReview = body
+                expect (updatedReview.votes).toBe(10)
+                expect(updatedReview.new_name).toBe(undefined)
+            })
+        });
+        describe('Errors for PATCH/api/reviews/:review_id', () => {
+            test('400: responds with 400 when sending body with missing fields ', () => {
+                return request(app)
+                .patch('/api/reviews/2')
```

Comment:
I would say that an empty send object with no inc_votes would just return the unchanged object with a 200 status, so wouldn't actually have this test....
However you can change this test to be something I would test for: 400 inc_votes wrong data type {inc_votes: 'not-a-number'}